src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -1189,12 +1183,31 @@ public object Get(System.Type entityClass, object id)
 		/// <param name="lockMode"></param>
 		/// <returns></returns>
 		public object Get(System.Type clazz, object id, LockMode lockMode)
+		{
+			return Get(clazz.FullName, id, lockMode);
+		}
+
+		/// <summary>
+		/// Return the persistent instance of the given entity name with the given identifier, or null
+		/// if there is no such persistent instance. (If the instance, or a proxy for the instance, is
+		/// already associated with the session, return that instance or proxy.)
+		/// </summary>
```

Comment:
```suggestion
		/// <inheritdoc />
```

Reply:
Hm.. I wonder if it would cause xml doc error as it's not yet in interface.. Will see...

Reply:
I do not think it will not cause an error, but will cause the member to have no documentation. I have overlooked it was of course not yet in the interface. Anyway as you wish here. I consider `Impl` namespace is not belonging to public APIs meant to be directly used by users.